### ISSUE TITLE
Cure poison quest line (Horde)

### ISSUE THREAD

**Issue by Hokraa** [dalaranwow/dalaran-wow#5507]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I am unable to begin the Cure Poison quest line, which should be available at level 14.  This is supposed to begin at the druid trainer in Thunderbluff.  The next step should be going to Moonglade, and then on to Crossroads, where you have to give salve to sick beasts.  I do not have any of these quests available to me and the Cure Poison spell does not appear to be available from any Druid trainer.

**Expected behaviour**: Tell us what should happen instead
Either the quests should be available from the appropriate quest givers, or the spell should be available from the trainer.  Neither option is available for me at this point (level 58).

**Steps to reproduce the problem**:

1.  I checked the quest start point in Thunderbluff.  No quest is available.
2.  I checked the further stages of the quest in Moonglade and Crossroads.  No quests available.
3. The Cure Poison is not availabel from the trainers.  (I have Abolish Poison, but Cure Poison is much better by saving global cool down when the target isn't poison as well as saving mana by not casting an unnecessary spell.)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
The quest can be completed and the spell can be obtained as intended. Make sure that you have completed all the pre-quests of the chain. Check WotLK sources for further info.

Closing
